engine/src/flutter/runtime/runtime_controller_unittests.cc

```diff
@@ -0,0 +1,151 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/runtime/runtime_controller.h"
+#include "flutter/runtime/runtime_delegate.h"
+
+#include "flutter/lib/ui/semantics/semantics_update.h"
+#include "flutter/shell/common/shell_test.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter::testing {
+
+fml::AutoResetWaitableEvent message_latch;
```

Comment:
This should be made a local, unless there's a reason that's impossible. If it's not possible, it should have static linkage.